src/components/Content/Content.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { items } from "../../mock/list.json";
+
+export default function Content() {    
+    return (
+        <div>
+            {items            
+            .map((item, index) => (              
+                <section key={index}>
```

Comment:
It would be better if you create a component for each Item, with its own classes